discovery_engine_core/web-api2/src/logging.rs

```diff
@@ -14,25 +14,31 @@
 
 //! Setup tracing on different platforms.
 
-use std::{fs::OpenOptions, path::Path, sync::Once};
+use std::{fs::OpenOptions, path::PathBuf, sync::Once};
 
+use serde::{Deserialize, Serialize};
 use tracing::Level;
 use tracing_subscriber::{
     filter::{LevelFilter, Targets},
     layer::SubscriberExt,
     util::SubscriberInitExt,
 };
 
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct Config {
+    pub(crate) file: Option<PathBuf>,
+}
+
 static INIT_TRACING: Once = Once::new();
```

Comment:
:question: Do we still need this? with an application should be easy to see that we are calling init_tracing only once in the main.

Reply:
Doesn't hurt to keep it, could avoid some wasted time in some integration testing situations.